src/entity/LightningBolt.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+declare(strict_types=1);
+
+namespace pocketmine\entity;
+
+use pocketmine\event\entity\EntityDamageEvent;
+use pocketmine\nbt\tag\CompoundTag;
+use pocketmine\network\mcpe\NetworkBroadcastUtils;
+use pocketmine\network\mcpe\protocol\AddActorPacket;
+use pocketmine\network\mcpe\protocol\types\entity\EntityIds;
+use pocketmine\network\mcpe\protocol\types\entity\PropertySyncData;
+use pocketmine\world\sound\ThunderSound;
+
+final class LightningBolt extends Entity{
+
+  private int $creationTick;
+
+	public static function getNetworkTypeId() : string{
+		return EntityIds::LIGHTNING_BOLT;
+	}
+
+	protected function getInitialSizeInfo() : EntitySizeInfo{
+		return new EntitySizeInfo(1.8, 0.6);
+	}
+
+	protected function getInitialGravity() : float{
+		return 0.0;
+	}
+
+	protected function getInitialDragMultiplier() : float{
+		return 0.02;
+	}
+
+	public function getName() : string{
+		return "Lightning Bolt";
+	}
+
+	protected function initEntity(CompoundTag $nbt) : void{
+	  $this->creationTick = $this->getWorld()->getServer()->getTick();
+		parent::initEntity($nbt);
+		$this->setNameTagVisible(false);
+		$this->setSilent();
+	}
+
+	public function onUpdate(int $currentTick) : bool{
+		if($currentTick - $this->creationTick > 20){
+			$this->flagForDespawn();
+			return false;
+		}
+
+		foreach($this->getWorld()->getNearbyEntities($this->getBoundingBox()->expandedCopy(3, 3, 3), $this) as $entity){
+			if($entity instanceof Living){
+				$entity->attack(new EntityDamageEvent($entity, EntityDamageEvent::CAUSE_LIGHTNING, 5.0));
+			}
+		}
+
+		return parent::onUpdate($currentTick);
+	}
+
+	public function spawnToAll() : void{
+		NetworkBroadcastUtils::broadcastPackets(
+			$this->getViewers(),
+			[
+				AddActorPacket::create(
+					$this->getId(),
+					$this->getId(),
+					self::getNetworkTypeId(),
+					$this->location,
+					null,
+					0.0,
+					0.0,
+					0.0,
+					0.0,
+					[],
+					[],
+					new PropertySyncData([], []),
+					[]
+				),
+			]
+		);
+
+		$this->getWorld()->addSound($this->location, new ThunderSound());
```

Comment:
This could be sent with a `sendSpawnPacket` override, or more likely `onFirstUpdate()`